www/src/pages/components/modal.js

```diff
@@ -24,7 +24,10 @@ export default withLayout(function ModalSection({ data }) {
         Static Markup
       </Heading>
       <p>A modal dialog component</p>
-      <ReactPlayground codeText={ModalStatic} />
+      <ReactPlayground
+        exampleClassName="bs-example-modals-static"
```

Comment:
i wonder if we should actually just have the modal dialog as a component... we had people confused by this before

Reply:
Funny thing, that is what we are doing in the example now. This bonus class isn't doing anything, just noticed today